### ISSUE TITLE
Insufficient contrast for $border-color-ui (a11y)

### ISSUE THREAD

**Issue by alexdahl-okta** [okta/odyssey#17]
As of July 2020,`$border-color-ui` is Gray 300 (`#c1c1c8`): https://github.com/okta/odyssey/blob/master/packages/odyssey/src/scss/abstracts/_tokens.scss#L92

This color is used as the only visual indicator for several components, such as Text Input and Checkbox.

<img width="477" alt="Screen Shot 2020-07-30 at 1 42 01 PM" src="https://user-images.githubusercontent.com/26014482/88984846-7cc6ef80-d26a-11ea-9353-3d53c2da4d8b.png">

WCAG AA requires user interface components to have a contrast ratio of at least 3:1 against adjacent colors: https://www.w3.org/TR/WCAG21/#non-text-contrast

Gray 300 fails this test against white.
I'm not sure if it should pass against other background colors as well.

**Comment by joejezowski-okta** [comment]
Thanks for flagging this @alexdahl-okta! Taking a peek and having a chat about it with Shred. I'll keep you posted on progress here.

**Comment by edburyenegren-okta** [comment]
@joejezowski-okta @arnoldsandoval-okta during our first round of colors, we landed on a `$border-color-ui` that met 3:1 contrast guidelines to address this. our current selection is back down to 1.79:1 instead of 3:1.

The lightest gray we have that meets 3:1 criteria is `gray, 600` as `gray, 500` is not available in code right now.

it feels odd to me to separate out `$border-color-ui` and `$border-color-display` if neither of them are compliant.

we can always keep both variables to make one compliant in the future if that becomes an option.

thoughts about how to address? if we should address?
